src/components/offers/HrOfferButtons.js

```diff
@@ -24,14 +24,16 @@ export const hrOfferButtons = [
             })
         },
         style: {
-            color: "secondary"
+            style: { backgroundColor: theme.status.danger.main, color: '#ffffff' },
```

Comment:
czy skoro mamy te kolorki to nie można by też tego ffffff zamienić na słowny kolorek?

Reply:
Prawdziwy frontendowiec widzi świat w hexie